### DISCUSSION TITLE
Spot Metering on boot/quick capture

### DISCUSSION THREAD

**Discussion by dutchct** [gopro/labs#489] [Q&A]
Hi guys,

Just wondering if it's possible to get a spot meter command working on boot with quick capture?

This command works on boot: !MBOOT="oSM50,59"

But I'm not seeing the spot metering brackets when using quick capture. Anyway to make this work?

**Comment by dnewman-gpsw** [comment]
Quickcapture happens before the BOOT command.  QC is that fast.  So the spot meter can't be enabled after the recording start.

**Reply by dutchct** [reply]
Thinking of a way around this, i wonder if I can use a second recording command on boot?

**Reply by dnewman-gpsw** [reply]
Sure, it just will not work with QuickCapture, but you can set the camera to record on regular boot, it just takes a couple of seconds more time.
